src/main/java/com/elytradev/concrete/inventory/gui/widget/WSlot.java

```diff
@@ -28,28 +28,23 @@
 
```

Comment:
Why rename this? Especially to WSlot, which reduces the amount of information the name conveys? I could see moving up to something like WItemSlotArea to indicate that this class is capable of representing many itemslots in the gui.